src/app/projectExamples/projectExamples.module.scss

```diff
@@ -59,44 +62,263 @@
 .processContainerTitleList {
 	display: flex;
 	font-weight: 300;
-	gap: 30px;
-	font-size: 24px;
-	line-height: 130%;
+	gap: 40px;
 }
 
 .clientWrap {
-	padding: 34px 0;
+	padding: 45px 0;
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
 	border-bottom: 1px solid var(--eerie-black);
 
 	&__title {
-		font-size: 24px;
 		font-weight: 300;
-		line-height: 130%;
 	}
 }
 
 .summaryWrapper {
-	padding: 34px 0 46px;
+	padding-top: 46px;
 }
 
 .summaryContainer {
 	&__titleContent {
```

Comment:
can you add display: block to summaryContainer selector here and in other similar places?